extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -663,7 +693,7 @@ public static function getRecommendations( $contentLanguage ) {
 	}
 
 	private static function fallbackToSupportedLanguages( $language ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)